### PR TITLE
Issue69 messages by organization

### PR DESCRIPTION
**Pull request by DannyPeck**
Added admin route to get all messages by organization.

### CODE REVIEW

**Comment by DannyPeck** [comment]
I will resolve the merge conflicts shortly.

File: app/controllers/MessageController.js [hunk 1]
```diff
@@ -14,6 +14,53 @@ function MessageController () {
...
+MessageController.prototype.create = function () {
+  var opts = {
+    on: {
+      preCreate: function (req, doc, callback) {
+        var token = req.headers.authorization.split(' ')[1];
+
+        User.findOne({token: token}, function (err, user) {
+          if (err) {
+            return callback(err);
+          }
```
Comment (cotz1995) [inline app/controllers/MessageController.js]: we totally should have made a wrapper for this haha

File: app/models/Message.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 var schema = new mongodb.Schema ({
-    sender_email:      {type: String, required: true, trim: true},
-    receiver_email:    {type: String, required: true, trim: true},
+    sender_email:   {type: String, required: true, trim: true},
+    receiver_email: {type: String, required: true, trim: true},
+    org_id:         {type: String, required: false, trim: true},
     received:       {type: Boolean, required: true},
     viewed:         {type: Boolean, required: true},
-    expireAt:        {type: Date, required: true},
+    expireAt:       {type: Date, required: false, default: Date.now},
```
Comment (cotz1995) [inline app/models/Message.js]: Have you tested this? Because I honestly don't remember if it's Date.now or Date.now()

Comment (DannyPeck) [inline app/models/Message.js]: I am actually worried about this, I was trying to get the post from the user portal to work and it didn't like expireAt.  Thank you for reminding me.

Comment (DannyPeck) [inline app/models/Message.js]: I will test if its Date.now or Date.now().

**Pull request merged**